recipe/meta.yaml

```diff
@@ -44,24 +30,24 @@ requirements:
     - python
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - numpy 1.9.*
+    - numpy 1.13.*
     - scipy
     # For tests.
     - pillow
-    
   run:
     - libgcc
     - python
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
-    - numpy >=1.9
+    - numpy >=1.13
     - scipy
+    - six
 
 test:
   source_files:
     # Test data
-    - boat.png
-    - lena.png           # [linux]
+    - extdata/boat.png
+    - extdata/lena.png           # [linux]
```

Comment:
nit: Would be nice to realign this selector with the one below.

Reply:
you mean realign the `# [linux]` part, right?